src/components/structures/MatrixChat.js

```diff
@@ -1061,6 +1061,7 @@ module.exports = React.createClass({
                     defaultIsUrl={this.getDefaultIsUrl()}
                     brand={this.props.config.brand}
                     teamServerConfig={this.props.config.teamServerConfig}
+                    queryParams={this.props.startingFragmentQueryParams}
```

Comment:
Works for me. `Registration` will need the `team_token` later, but that could just be another prop